### ISSUE TITLE
atgc-bundle-wind-turbine: create a wind turbine

### ISSUE THREAD

**Issue by jbilcke** [autogica/atgc#27]
can be bought and used to generate energy

quite logically the higher located on the map (heightmap terrain), the more energy it will yield

it's not just for fun, actually this is just a cool way of experimenting with hill exploration algorithms in the game, so robots will have to find the most energy-efficient location, ie. try to reach an equilibrium between the energy necessary to climb the hill and the energy that will be returned etc.. while constantly dealing with hostile threats, changes in the terrain like new obstacles etc..

so basically we need a simple 3D model of some kind of tall pylon and blades
